PluginTemplateProject/pom.xml

```diff
@@ -131,6 +131,11 @@
                 <groupId>org.scala-tools</groupId>
                 <artifactId>maven-scala-plugin</artifactId>
                 <version>2.15.2</version>
+                <configuration>
+                    <excludes>
+                        <exclude>com/alpine/templates/db/*</exclude>
+                    </excludes>
+                </configuration>
```

Comment:
But they still have to decide whether to add it to plugins.xml. I think it would be less confusing for them to just decide to remove it from there.